spikeinterface/widgets/matplotlib/unit_summary.py

```diff
@@ -0,0 +1,68 @@
+import numpy as np
+
+from ..base import to_attr
+from ..unit_summary import UnitSummaryWidget
+from .base_mpl import MplPlotter
+
+
+from .unit_locations import UnitLocationsPlotter
+from .amplitudes import AmplitudesPlotter
+from .unit_waveforms import UnitWaveformPlotter
+from .unit_waveforms_density_map import UnitWaveformDensityMapPlotter
+
+from .autocorrelograms import AutoCorrelogramsPlotter
+
+
+class UnitSummaryPlotter(MplPlotter):
+
+    def do_plot(self, data_plot, **backend_kwargs):
+
+        dp = to_attr(data_plot)
+        
+        # force the figure without axes
+        if 'figsize' not in backend_kwargs:
+            backend_kwargs['figsize'] = (18, 7)
+        backend_kwargs = self.update_backend_kwargs(**backend_kwargs)
+        backend_kwargs['num_axes'] = 0
+        backend_kwargs['ax'] = None
+        backend_kwargs['axes'] = None
+        
+        self.make_mpl_figure(**backend_kwargs)
+        
+        # and use custum grid spec
+        fig = self.figure
+        gs = fig.add_gridspec(3, 4)
```

Comment:
In my opinion you should create the gridspec based on `dp.plot_data_unit_locations`, `dp.plot_data_acc`, and `dp.plot_data_amplitudes`

Reply:
@samuelgarcia what about this? Otherwise it will leave blank spaces..

Reply:
I feel lazy about this but I will try somthing.

Reply:
Done